docs/docsite/rst/scenario_guides/guide_gce.rst

```diff
@@ -254,3 +254,43 @@ Note that use of the "add_host" module above creates a temporary, in-memory grou
 in the 'new_instances' group, if so desired.  Any sort of arbitrary configuration is possible at this point.
 
 For more information about Google Cloud, please visit the `Google Cloud website <https://cloud.google.com>`_.
+
+Migration Guides
+----------------
+
+gce.py -> gcp_compute_instance.py
+`````````````````````````````````
+We're encouraging everyone to move from the `gce` module to the
+`gcp_compute_instance` module. The new `gcp_compute_instance` module has better
```

Comment:
Here and below, I'd remove the word "new" once you've added the relevant versions. The module will stop being "new" before the migration instructions stop being useful.